internal/security/secretstoreclient/vault.go

```diff
@@ -156,3 +155,79 @@ func (vc *vaultClient) CreateToken(token string, parameters map[string]interface
 		ResponseObject:       response,
 	})
 }
+
+func (vc *vaultClient) ListAccessors(token string, accessors *[]string) (statusCode int, err error) {
+	var response ListTokenAccessorsResponse
+	code, err := vc.doRequest(commonRequestArgs{
+		AuthToken:            token,
+		Method:               "LIST",
+		Path:                 ListAccessorsAPI,
+		JSONObject:           nil,
+		BodyReader:           nil,
+		OperationDescription: "list token accessors",
+		ExpectedStatusCode:   http.StatusOK,
+		ResponseObject:       &response,
+	})
+	*accessors = response.Data.Keys
+	return code, err
+}
+
+func (vc *vaultClient) RevokeAccessor(token string, accessor string) (statusCode int, err error) {
+	parameters := RevokeTokenAccessorRequest{Accessor: accessor}
+	return vc.doRequest(commonRequestArgs{
+		AuthToken:            token,
+		Method:               http.MethodPost,
```

Comment:
not `http.MethodDelete`?

Reply:
Correct.  See https://www.vaultproject.io/api/auth/token/index.html#revoke-a-token-accessor.  Revoke by accessor is defined to be POST.

Reply:
ok